app/views/sign_requests/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, "Request signs") %>
+
+<div class="grid-x list">
+  <div class="list__section cell grid-x">
+    <div class="cell small-12 large-9 margin-bottom-2">
+      <h1>Request signs</h1>
+    </div>
+
+    <div class="cell small-12 large-6">
+      <h3 class="margin-bottom-1">Request signs on Facebook</h3>
+
+      <p>
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+        Intro and overview information on Facebook requests, and possible future integration into this website.
+      </p>
+    </div>
+
+    <div class="cell small-12 large-6">
+      <div class="row align-right align-middle">
+        <%= inline_svg_pack_tag "media/images/facebook-light.svg", class: "margin-right-1", aria: true, title: "Facebook Logo" %>
+        <%= link_to "https://www.facebook.com/groups/213136893357675/", class: "button--with-icon" do %>
```

Comment:
This should probably be in configuration somewhere

Reply:
Do you mean as an env variable that is loaded in a config file?

Reply:
Yes - I'm not a fan of hardcoded contact details like this, but given the purpose of the page this may be OK